c7n/resources/elb.py

```diff
@@ -106,6 +108,46 @@ def get_resources(self, resource_ids):
                 return []
             raise
 
+    def get_elbs_from_result_page(self, client, rp):
+        elb_descriptions = rp['LoadBalancerDescriptions']
+        self.add_tags_to_results(client, elb_descriptions)
+        return elb_descriptions
+
+    def add_tags_to_results(self, client, elbs):
+        """
+        Gets the tags for the ELBs and adds them to
+        the result set.
+        """
+        elb_names = [elb['LoadBalancerName'] for elb in elbs]
+        names_to_tags = {}
```

Comment:
don't really think you need the names_to_tag map, you could just chunk the elbs and let the process_tags function directly update the elb, don't really need the result as its mutating a shared ref with transient ownership, just the exception processing.

that said there's requests for this now, so happy to cleanup after the fact post merge.